File: Day_21/Q82.java
import java.util.*;
class Q82{
    //Reversing The String
    static void reverse_string(StringBuilder str){
        int length = str.length();
        for(int i = 0;i < length/2;i++){
            char temp = str.charAt(length - i - 1);
            str.setCharAt((length - i -1), str.charAt(i));
            str.setCharAt(i,temp);
        }
    }
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        String str = sc.nextLine();
        StringBuilder sb = new StringBuilder(str);
        //Printing The Result
        reverse_string(sb);
        str = sb.toString();
        System.out.print(str);
        sc.close();
    }
}

File: Day_21/Q82.cpp
#include <bits/stdc++.h>
using namespace std;
//Reversing The String
void reverse_string(string &str){
    int length = str.length();
    for(int i = 0;i < length/2;i++){
        swap(str[i],str[length-i-1]);
    }
}
int main(){
    string str;
    cin >> str;
    reverse_string(str);
    //Printing The Result
    cout << str;
    return 0;
}